wondrous-app/pages/organization/[username]/boards.tsx

```diff
@@ -634,6 +638,11 @@ const BoardsPage = () => {
         : null
     );
   }
+
+  useHotkeys('tab+t', () => {
```

Comment:
my opinion is that we shouldn't do it at this level, but instead at this file 
`components/organization/wrapper/wrapper.tsx` triggering the `toggleCreateFormModal` function

Reply:
otherwise all good!